docs-guides/source/convert-pytorch-workflow.md

```diff
@@ -24,7 +24,20 @@ import torchvision
 torch_model = torchvision.models.mobilenet_v2(pretrained=True)
 # Set the model in evaluation mode.
 torch_model.eval()
+```
+
+```{admonition} Set the Model to Evaluation Mode
+
+To ensure that operations such as dropout are disabled, it's important to set the model to evaluation mode (_not_ training mode) before tracing. This setting also results in a more optimized version of the model for conversion.
+```
+
+## Generate a PyTorch Graph Version
```

Comment:
"Generate a PyTorch Graph Version" : that's an odd title, what does "version" mean?  
I think "Generate the PyTorch Graph"  would make more sense. 

(I know this is what it currently says, and I actually didn't realize it. Well whatever page you touch, you should question it, if its makes sense or not and look to improve it if it doesn't. Existing doc is not perfect. This is a good example of that. )

Reply:
Rephrased to `Capture the PyTorch Graph`